samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs

```diff
@@ -45,40 +45,6 @@ protected internal StorageAccountOperations(OperationsBase options, ResourceIden
         /// <summary> Gets the valid resource type for the operations. </summary>
         protected override ResourceType ValidResourceType => ResourceType;
 
-        /// <inheritdoc />
-        public async override Task<Response<StorageAccount>> GetAsync(CancellationToken cancellationToken = default)
-        {
-            using var scope = _clientDiagnostics.CreateScope("StorageAccountOperations.GetProperties");
-            scope.Start();
-            try
-            {
-                var response = await _restClient.GetPropertiesAsync(Id.ResourceGroupName, Id.Name, null, cancellationToken).ConfigureAwait(false);
-                return Response.FromValue(new StorageAccount(this, response.Value), response.GetRawResponse());
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
-        /// <inheritdoc />
-        public override Response<StorageAccount> Get(CancellationToken cancellationToken = default)
-        {
-            using var scope = _clientDiagnostics.CreateScope("StorageAccountOperations.GetProperties");
-            scope.Start();
-            try
-            {
-                var response = _restClient.GetProperties(Id.ResourceGroupName, Id.Name, null, cancellationToken);
-                return Response.FromValue(new StorageAccount(this, response.Value), response.GetRawResponse());
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
```

Comment:
If we are removing this method, then I think we should pass `StorageAccountExpand? expand` parameter as optional, like [rest client's method](https://github.com/Azure/autorest.csharp/blob/8efc2364091e61ced50cec9c9305a8e5aa063d9c/samples/Azure.Management.Storage/Generated/RestOperations/StorageAccountsRestOperations.cs#L309).

Example: 
```csharp
public async virtual Task<Response<StorageAccount>> GetAsync(StorageAccountExpand? expand = null, CancellationToken cancellationToken = default)
```

Reply:
that already exists, before we had 2 overloads, now we only have the one which is the one you mentioned on line 70

Reply:
Yes, the get method overload exists, but it takes nullable parameter(StorageAccountExpand? expand) but it should take nullable and optional parameter(StorageAccountExpand? expand = null) just like rest client method.

Reply:
Yes, the get method overload exists, but it takes nullable parameter(StorageAccountExpand? expand), now that we removed this get method, user has to always pass something in expand  parameter. To fix this, it should take nullable and optional parameter(StorageAccountExpand? expand = null) just like rest client method.